web/lib/components/home/stats.module.css

```diff
@@ -0,0 +1,12 @@
+.row {
+  @apply grid grid-cols-2;
+  grid-template-areas: 'word count';
+}
+
+.word {
+  grid-area: word;
+}
+
+.count {
+  grid-area: count;
+}
```

Comment:
No need to define this CSS because vanilla classes as follows are good enough.
```jsx
<div key={`${db}-${count}`} className="text-sm font-bold grid grid-cols-2">
  <span>{db}</span>
  <span className="text-sm font-thin">
    {count} {_e('common.words')}
  </span>
</div>
```